code/src/components/Todo.js

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useDispatch } from 'react-redux';
+import { todos } from 'reducers/todos';
+
+const moment = require('moment');
```

Comment:
This is a commonjs import. Since we have es6 all set up, you can write it like the other imports:

```js
import moment from 'moment'
```